src/Components/Blazor/WebAssembly.Authentication/test/WebAssemblyAuthenticationServiceCollectionExtensionsTests.cs

```diff
@@ -0,0 +1,256 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using Microsoft.AspNetCore.Blazor.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Options;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Components.WebAssembly.Authentication
+{
+    public class WebAssemblyAuthenticationServiceCollectionExtensionsTests
+    {
+        [Fact]
+        public void CanResolve_AccessTokenProvider()
+        {
+            var builder = WebAssemblyHostBuilder.CreateDefault();
+            builder.Services.AddApiAuthorization();
```

Comment:
Neither the name of the test class nor test implies this is about "API authorization". Should we have more a specific name for the class or method here?